src/api/publicationApi.ts

```diff
@@ -28,7 +28,7 @@ export const createNewPublicationFromDoi = async (url: string, owner: string, di
 };
 
 export const createNewPublication = async (publication: Publication, dispatch: Dispatch) => {
-  const url = `${API_URL}/insert-resource`;
+  const url = '/insert-resource';
```

Comment:
stemmer for å lage navngitte const og exportere disse. Vi kunne brukt dem for cypress også